edb/edgeql/parser/grammar/sdl.py

```diff
@@ -624,6 +624,7 @@ def reduce_CreateRegularProperty(self, *kids):
             target=kids[3].val,
             commands=kids[4].val,
         )
+        # reject_unqualified(self.val)
```

Comment:
?

Should this be uncommented?

Reply:
Yes, that generates a few more test failures that I'm working on now. I commented those for now so we can focus on the DROP REQUIRED test failures on CI.